app.rb

```diff
@@ -4,128 +4,40 @@
 require './book'
 require './classroom'
 require './rental'
+require './lib/create_books'
+require './lib/create_persons'
+require './lib/create_rental'
+require './lib/displayer'
 
 class App
   def initialize
     @books = []
     @persons = []
     @rentals = []
-  end
-
-  def start_console
-    puts 'Welcome to my School Library!'
-    until list_of_options
-      input = gets.chomp
-      if input == '7'
-        puts 'Thank you for using my School Library!'
-        break
-      end
-
-      option input
-    end
+    @displayer = Displayer.new
   end
 
   def list_all_books
-    puts 'Database is empty! Add a book.' if @books.empty?
-    @books.each { |book| puts "[Book] Title: #{book.title}, Author: #{book.author}" }
+    @displayer.display_books(@books)
   end
 
   def list_all_persons
-    puts 'Database is empty! Add a person.' if @persons.empty?
-    @persons.each { |person| puts "[#{person.class.name}] Name: #{person.name} Age: #{person.age} Id: #{person.id}" }
+    @displayer.display_persons(@persons)
   end
 
   def create_person
-    print 'To create a student, press 1, to create a teacher, press 2 :'
```

Comment:
- [ ] Asking the user for the initial choice of options( or to quit) should have been done in the main.rb file. Please let's fix this.

Reply:
These are the deleted code